apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/environment/EnvTest.java

```diff
@@ -97,4 +97,71 @@ public void getName() {
         Env.addEnvironment(name);
         assertEquals(name.trim().toUpperCase(), Env.valueOf(name).toString());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void transformEnvBlankTest() {
+        assertEquals(Env.UNKNOWN,Env.transformEnv(""));
+        assertEquals(Env.UNKNOWN,Env.transformEnv(null));
+        assertEquals(Env.UNKNOWN,Env.transformEnv("   "));
+    }
+
+    @Test
+    public void transformEnvSpecialCaseTest() {
+        // Prod/Pro
+        assertEquals(Env.PRO,Env.transformEnv("prod"));
+        assertEquals(Env.PRO,Env.transformEnv("PROD"));
+
+        //FAT/FWS
+        assertEquals(Env.FAT,Env.transformEnv("FWS"));
+        assertEquals(Env.FAT,Env.transformEnv("fws"));
+    }
+
+    @Test
+    public void transformEnvNotExistTest() {
+        assertEquals(Env.UNKNOWN,Env.transformEnv("notexisting"));
+        assertEquals(Env.LOCAL,Env.transformEnv("LOCAL"));
+    }
+
+    @Test
+    public void transformEnvValidTest() {
+        assertEquals(Env.UNKNOWN,Env.transformEnv("UNKNOWN"));
+        assertEquals(Env.LOCAL,Env.transformEnv("LOCAL"));
+        assertEquals(Env.FAT,Env.transformEnv("FAT"));
+        assertEquals(Env.FAT,Env.transformEnv("FWS"));
+        assertEquals(Env.PRO,Env.transformEnv("PRO"));
+        assertEquals(Env.PRO,Env.transformEnv("PROD"));
```

Comment:
Add some blank char like space or tab in the string.

```suggestion
        assertEquals(Env.PRO,Env.transformEnv("PROD"));
        assertEquals(Env.PRO,Env.transformEnv("PROD "));
        assertEquals(Env.PRO,Env.transformEnv("\tPROD\t"));
```